### DISCUSSION TITLE
hugo-blog-embed-mastodon/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#145] [Giscus]
# hugo-blog-embed-mastodon/

本文介绍了如何将 Hugo 博客集成到 Mastodon 中。作者首先解释了 API 和 RSS 两种方式可以用来集成 Mastodon，然后详细介绍了使用 API 和 JS 的方法。他使用了一个名为 mastodon-embed-timeline 的 JS 包进行集成，并提供了安装步骤。接下来，作者说明了如何创建一个新的模板文件并修改其中的 CSS 和 JS 引入部分以及配置 Mastodon 参数。最后，作者指导读者在 content 目录下创建一个页面，并插入渲染 Mastodon 容器所需的 HTML 代码。文章给出了具体示例和技巧，帮助读者实现 Hugo 博客与 Mastodon 的集成。

https://www.eallion.com/hugo-blog-embed-mastodon/

**Comment by wangyunzi** [comment]
最近memos频繁升级出现问题都准备考虑换个频道了，比如mastodon

**Reply by eallion** [reply]
😆 好像是前段时间有 2 次 Breaking Change，现在倒是稳定了。Mastodon 有点重，可以试试 GoToSocial 或者 Misskey。
